shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -18,107 +18,55 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol FlutterPluginRegistrar;
 @protocol FlutterPluginRegistry;
 
-/**
```

Comment:
Man GitHub diffing is pretty crappy. But I'm assuming you're just copy pasting stuff around.

Reply:
Yep, just moved around.